### PR TITLE
add: missing mocha dev dependency

### PR DESCRIPTION
**Pull request by zhuangya**
- add missing mocha dependency
- travis settings and badge.

### CODE REVIEW

**Comment by zhuangya** [comment]
ta-da

**Comment by zhuangya** [comment]
also, remember to turn travis on for this project.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+language: node_js
+node_js:
+  - "4.0"
+  - "4.1"
```
Comment (yorkie) [inline .travis.yml]: Why 4.1? Suggest to add LTS version as well

Comment (zhuangya) [inline .travis.yml]: `4.1.x` just another stable release



> Suggest to add LTS version as well

hmm, make sense. doing it right now.

Comment (zhuangya) [inline .travis.yml]: #2

**Pull request merged**